cleanhttp.go

```diff
@@ -56,6 +71,6 @@ func DefaultClient() *http.Client {
 // for clients that will be re-used for the same host(s).
 func DefaultPooledClient() *http.Client {
 	return &http.Client{
-		Transport: DefaultPooledTransport(),
+		Transport: DefaultPooledTransportWithMin(10),
```

Comment:
i have changed the default behavior to 10, however, for backward compatibility I can go with 0 as well.